src/main/java/com/github/fishio/settings/Settings.java

```diff
@@ -66,7 +66,9 @@ public SimpleDoubleProperty getSliderProperty(String setting) {
 		SimpleDoubleProperty s = sliderSettings.get(setting);
 		if (s == null) {
 			logger.log(LogLevel.ERROR, "Setting '" + setting + "' not found!");
```

Comment:
*Auto-generated comment*

    
    The String literal "Setting '" appears 4 times in this file; the first occurrence is on line 68